Rubberduck.Parsing/ComReflection/ComStruct.cs

```diff
@@ -27,14 +27,14 @@ public ComStruct(ITypeLib typeLib, ITypeInfo info, TYPEATTR attrib, int index)
 
         private void GetFields(ITypeInfo info, TYPEATTR attrib)
         {
-            var names = new string[255];
+            var names = new string[1];
             for (var index = 0; index < attrib.cVars; index++)
             {
                 IntPtr varPtr;
                 info.GetVarDesc(index, out varPtr);
                 var desc = (VARDESC)Marshal.PtrToStructure(varPtr, typeof(VARDESC));
                 int length;
-                info.GetNames(desc.memid, names, 255, out length);
+                info.GetNames(desc.memid, names, 1, out length);
```

Comment:
shouldn't 1 be `names.Length` here? (and in ComModule)?
Also why is there no corresponding change in ComEnumerationMember?

Reply:
Yep, I'll do that.   For ComEnumerationMember, the array was declared unnecessarily large, but the call to GetNames was correct.   I'll change it to names.Length there as well though.